index.html

```diff
@@ -10861,9 +10861,74 @@ <h2>A project for first-time contributions</h2>
             <!-- ^^^^ make sure there is one line of space between your card start and the card end above it ^^^^ -->
             <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
             <!-- ^^^^ ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ^^^^ -->
-
-
-
+<!-- ________ Contributor's card START ________  -->
+            <div class="card">
+                <p class="name">Name</p>
+                <p class="contact">
+                    <i class="fab fa-twitter"></i>
+                    <a href="YOUR twitter account link" target="_blank">@yourHandle</a>
+                </p>
+                <p class="about">
+                    Write here one sentence about yourself. Anything you want: Why do you program, a fun fact, why
+                    do you like programming, what you hope to achieve, your pet's name...
+                </p>
+                <div class="resources">
+                    <p>3 Useful Dev Resources</p>
+                    <ul>
+                        <li>
+                            <a href="#" target="_blank" title="This is a link to an awesome resource">
+                                Link to resources
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank"
+                                title="This is another link to web development tool I use a lot">
+                                Link to resources
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" target="_blank" title="The third link to something useful for developers">
+                                Link to resources
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+            <!-- ________ Contributor's card END ________  -->
+            
+            <!-- ________ Contributor's card START ________  -->
```

Comment:
Your card is not in the right place. It needs to be placed directly above the line that contains
```
<!-- ^^^^ Paste YOUR CARD directly ^^ ABOVE ^^ this line ^^^^ -->
```
You will find this line just above the place where your code changes start.